collections.sh

```diff
@@ -1,2 +1,30 @@
 #!/bin/bash
 set -e
+
+
+ecr_create()
+{
+  ECR_REPO=$1
+
+  curl -f -L "url"  --request GET "$LAZY_API_URL/profiles/prod/regions/$AWS_REGION/ecr/repo/$ECR_REPO/repo-policy" \
+   --header "x-api-key:  $LAZY_API_KEY" \
+  || { 
+    wget \
```

Comment:
Can you do an echo here "Repo not found. Hence creating a new repo with name ${ECR_REPO}"

Reply:
Added